merchant-sdk/example-app/src/main/java/net/gini/android/merchant/sdk/exampleapp/MainViewModel.kt

```diff
@@ -38,9 +38,10 @@ class MainViewModel(
         ++currentIndex
     }
 
-    fun setDocumentForReview(documentId: String) {
```

Comment:
I think this can be removed when merging with EC-60 - since the instantiation of the container fragment contains the documentId, setting the document for review can be done internally.

Reply:
Thank you, I will delete it then completely.